packages/passport/sdk/src/errors/handleTrackError.ts

```diff
@@ -0,0 +1,10 @@
+import { trackError } from '@imtbl/metrics';
+
+const moduleName = 'passport';
+
+export const handleError = (method: string, error: unknown) => {
+  const wrappedError = new Error(`${moduleName} ${method} error`, { cause: error });
+
+  trackError('passport', method, wrappedError);
```

Comment:
Given this is called within a `withPassportError` arent we already calling `trackError` there?

Reply:
I put it in both places to see which would be preferable. If Passport team is ok with modifying the `withPassportError` function, then I'll remove the `handleError` function from the transfer methods.

Reply:
Ah ok, I think it makes sense to keep it all contained within `withPassportError` since that does all the error handling already